crates/ruma-common/src/events/image.rs

```diff
@@ -0,0 +1,333 @@
+//! Types for extensible image message events ([MSC3552]).
+//!
+//! [MSC3552]: https://github.com/matrix-org/matrix-spec-proposals/pull/3552
+
+use js_int::UInt;
+use ruma_macros::EventContent;
+use serde::{Deserialize, Serialize};
+
+use super::{
+    file::{EncryptedContent, FileContent, FileContentInfo},
+    message::{MessageContent, Text},
+    room::message::Relation,
+};
+use crate::MxcUri;
+
+/// Information about a thumbnail file content.
+#[derive(Clone, Debug, Default, Serialize, Deserialize)]
+#[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
+pub struct ThumbnailFileContentInfo {
+    /// The mimetype of the thumbnail, e.g. “image/png”.
```

Comment:
I think an inline code tag makes more sense here than quotation marks. I'd also be okay with ascii quotation marks, but not whatever these are 😄 

```suggestion
    /// The mimetype of the thumbnail, e.g. `image/png`.
```